data/fixtures/scopes/python/collectionItem.unenclosed.iteration3.scope

```diff
@@ -2,10 +2,25 @@ def foo():
     global bar, baz
 ---
 
-[Range] = 1:11-1:19
+[#1 Range] =
+[#1 Domain] = 0:8-0:8
+          ><
+0| def foo():
+
+
+[#2 Range] = 1:4-1:19
+      >---------------<
+1|     global bar, baz
+
+[#2 Domain] = 1:0-1:19
+  >-------------------<
+1|     global bar, baz
+
+
+[#3 Range] = 1:11-1:19
```

Comment:
NB: this is the old range from the language server, it's not marked as such, but you know because it's identical to the old code